Flipper/Packages/UI/Sources/Archive/Info/Components/InfraredMenu.swift

```diff
@@ -17,6 +23,12 @@ struct InfraredMenu: View {
                     isPresented = false
                     onEdit()
                 }
+                .foregroundColor(
+                    isEmulating
```

Comment:
Would be better to extract the button to it's own view and use isEnabled to choose the color